tools/sigclient/pkg/query/query.go

```diff
@@ -926,15 +926,15 @@ func runBenchmarkQueryBatch(uuidList []string, wg *sync.WaitGroup, semaphore cha
 			"queryLanguage": "Splunk QL",
 		}
 		evaluationType := "total"
-		relation := "gt"
-		expectedValue := "0"
+		var relation, expectedValue string
+		relation = "eq"
+		expectedValue = "1"
```

Comment:
So now that we're using a random seed, we'll always get distinct UUIDs, even when run with parallelization—am I understanding that right?

Reply:
yes, now we can now check for equality condition